### PR TITLE
hd-Himanshu (PR)

### PR DESCRIPTION
**Pull request by Gem-Himanshu-Panchal**
Hi @gem-pawandeep, please review my code.

### CODE REVIEW

File: src/test/java/com/qa/helpdesk/stepdefinition/HomePage.java [hunk 1]
```diff
@@ -0,0 +1,1285 @@
...
+    }
+
+
+    //CREATE NEW TICKET
+    @And("^Check if \"(.*)\", \"(.*)\", \"(.*)\" and \"(.*)\" fields are present$")
+    public void checkIfFieldsArePresent(String subject, String type, String description, String department) {
+        passed = false;
+        String missingField = null;
+        try {
+            if (DriverAction.isExist(CommonLocators.elementById("input", subject))) {
```
Comment (gem-pawandeep) [inline src/test/java/com/qa/helpdesk/stepdefinition/HomePage.java:246]: remove this so many if else use & operator here

**Review by gem-pawandeep — CHANGES_REQUESTED** [review]
1. remove static waits.
2. remove System.out.println statements.
3. remove so many if else statements use AND condition.

File: src/test/java/com/qa/helpdesk/stepdefinition/HomePage.java (hunk 1 above)
Comment (Gem-Himanshu-Panchal) [reply src/test/java/com/qa/helpdesk/stepdefinition/HomePage.java:246]: Okay using && operator in single if condition

Comment (gem-pawandeep) [reply src/test/java/com/qa/helpdesk/stepdefinition/HomePage.java:246]: yes

Comment (Gem-Himanshu-Panchal) [reply src/test/java/com/qa/helpdesk/stepdefinition/HomePage.java]: Working on dynamic waits, rest all the changes are done.

**Review by gem-pawandeep — APPROVED** [review]
looks good!!!

**Review by gem-pawandeep — CHANGES_REQUESTED** [review]
1. Please remove all the statics waits it impacts code speed.
2. remove dead code.

**Pull request closed without merging**